internal-services/catalog/create-advisory-task.yaml

```diff
@@ -58,19 +61,20 @@ spec:
               key: git_author_email
       script: |
           #!/usr/bin/env sh
-          set -ox pipefail
+          set -eox pipefail
```

Comment:
The `x` should probably be removed as it will leak the `ACCESS_TOKEN` when using the env var in the git functions.

Reply:
fixed, thanks